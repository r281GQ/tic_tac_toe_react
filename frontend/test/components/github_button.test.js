import React from 'react';
import { shallow } from 'enzyme';
import GithubButton from './../../src/components/github_button';

describe('GithubButton test', () => {
  it('should render', () => {
    const component = shallow(<GithubButton />);
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
