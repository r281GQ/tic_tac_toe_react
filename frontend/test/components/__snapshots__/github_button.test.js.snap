// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GithubButton test should render 1`] = `
<IconButton
  disableTouchRipple={false}
  disabled={false}
  href="https://github.com/r281GQ/tic_tac_toe_react"
  iconStyle={Object {}}
  target="_blank"
  tooltip="Github"
  tooltipPosition="bottom-center"
  touch={false}
>
  <FontIcon
    className="fa fa-github white"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  />
</IconButton>
`;
